src/App.css

```diff
@@ -0,0 +1,32 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&family=Norican&family=Poppins&family=Roboto&display=swap');

```

Comment:
- [ ] I can see there is no mobile implementation:

![image](https://user-images.githubusercontent.com/57421823/227379116-83070472-dc73-43a9-a14e-f53d62c04204.png)

![image](https://user-images.githubusercontent.com/57421823/227379153-403547c7-9d0d-4173-9323-b7a352b1a840.png)

This is a core requirement, kindly implement some styling on the mobile view for your frontend project.

Reply:
Added style for mobile view